lib/matplotlib/backends/backend_pdf.py

```diff
@@ -20,14 +19,10 @@
 
 import numpy as np
 from six import unichr
-from six import BytesIO
+from io import BytesIO
 
```

Comment:
According to https://pythonhosted.org/six/ six.BytesIO is StringIO.StringIO in Python 2 not io.BytesIO does that make any difference?